src/state/collectibles/hooks.ts

```diff
@@ -15,7 +15,7 @@ export const useGetCollectibles = () => {
 
   useEffect(() => {
     // Fetch nfts only if we have not done so already
-    if (!isInitialized) {
+    if (!isInitialized && account) {
```

Comment:
At the moment on the `nft-market` - `/collectibles` crashes.

In quite a few places - we're calling actions that require an account, when the app doesn't yet have one. I've added these checks in a few places.

Reply:
I've broken out a lot of this work here https://github.com/pancakeswap/pancake-frontend/pull/2163/files